datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -3188,6 +3213,11 @@ type Owner {
     Source information for the ownership
     """
     source: OwnershipSource
+
+    """
+    Reference back to the owned urn for tracking purposes e.g. when sibling nodes are merged together
+    """
+    associatedUrn: String
```

Comment:
same question here (and all the ones below)

Reply:
yes good point